en/flight_stack/controller_diagrams.md

```diff
@@ -25,7 +25,7 @@ The diagrams use the standard [PX4 notation](../contribute/notation.md) (and eac
 
 * K-PID controller. See [Rate Controller](../config_mc/pid_tuning_guide_multicopter.md#rate-controller) for more information.
 * The integral authority is limited to prevent wind up.
-* The outputs are limited (in the mixer), usually at -1 and 1. 
+* The outputs are limited (in the mixer), usually at -1 and 1.
```

Comment:
Again, can I use the term "in the mixer"?

Reply:
Here I'd use control allocation module, as it refers to the implementation.